operator/src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -616,13 +613,18 @@ protected Step action(String ns) {
   private static class NamespaceRulesReviewStep extends Step {
     private final String ns;
 
+    NamespaceRulesReviewStep() {
+      this(null);
+    }
+
     NamespaceRulesReviewStep(String ns) {
       this.ns = ns;
     }
 
     @Override
     public NextAction apply(Packet packet) {
-      NamespaceStatus nss = namespaceStatuses.computeIfAbsent(ns, (key) -> new NamespaceStatus());
+      NamespaceStatus nss = namespaceStatuses.computeIfAbsent(
+          ns != null ? ns : operatorNamespace, (key) -> new NamespaceStatus());
```

Comment:
i feel this syntax could lead to maintenance issues as people wonder what exactly your intent was. 
not a big deal, but i would prefer to either add a comment, or just make it a little more explicit

Reply:
I'll add a comment